benchmark/src/sql/tables.sql

```diff
@@ -10,7 +10,7 @@ create table benchmark
 	description varchar(1000) not null,
 	run_tag varchar(10) not null,
 	config_tag varchar(50) not null,
-	node varchar(50) not null,
```

Comment:
GCP provides internal DNS for vms, so we use long names for the nodes when we form the cluster. These names did not fit the original 50 character limit.